src/components/post/PostBlock.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../../styles/post/PostBlock.scss";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import PostDropDown from "./PostDropDown";
+import moreIcon from "../../images/more-icon.svg";
+
+class PostBlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      username, content, imageSrc, postTime,
+    } = this.props;
+    return (
+      <div className="post-block">
+        <div className="post-info">
+          <div className="post-username">{username}</div>
+          <DropdownButton
+            id="post-more-button"
+            title={<img id="post-more-icon" src={moreIcon} alt="more-icon" />}
+            drop="down"
+            alignRight
+          >
+            <PostDropDown />
+          </DropdownButton>
+
+          <div className="post-time">{postTime}</div>
+        </div>
+        <img className="post-img" src={imageSrc} alt="more-icon" />
+        <div className="post-content">{content}</div>
+      </div>
+    );
+  }
+}
+
+PostBlock.propTypes = {
+  username: PropTypes.string.isRequired,
+  postTime: PropTypes.string.isRequired,
```

Comment:
Forgot about this in the initial review. Should we make this component take in a Date or Moment.js object and then we can handle the conversion to a string like "10 hours ago" in this component itself instead of passing it in as a string. This way we can just pass the data we get from the database straight to the component and it can handle everything instead of doing data->string conversion outside this component.